src/App/App.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { BrowserRouter, Routes, Route, Navigate } from 'react-router-dom';
+
+import { RepoPage } from './pages/RepoPage';
+import { Repositories } from './pages/Repositories';
+
+function App() {
```

Comment:
везде лучше использовать стрелочные функции, они занимают меньше места
```suggestion
const App = () => {
```